extensions/amp-access-fewcents/validator-amp-access-fewcents.protoascii

```diff
@@ -0,0 +1,27 @@
+tags: {  # amp-access-fewcents
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-access-fewcents"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-access-fewcents>
```

Comment:
I don't think we actually need this?

Reply:
> Thanks for breaking this up! Just a few comments.

Thanks @calebcordry for your review, I will make the changes you asked and let you know.

Reply:
Could you please help me here, I haven't understood why this file is required?

Reply:
its required for the validator to recognize a new extension e.g. `<amp-access-fewcents>` but since this is just a extension of the `<amp-access>` component I think we can just delete this file.

Reply:
Okay I will delete it.